cdm/core/src/main/java/ucar/nc2/ft/point/ProfileFeatureImpl.java

```diff
@@ -31,9 +34,9 @@ public ProfileFeatureImpl(String name, CalendarDateUnit timeUnit, String altUnit
     }
   }
 
-  public ProfileFeatureImpl(String name, String timeName, CalendarDateUnit timeUnit, String altName, String altUnits,
-      double lat, double lon, double time, int nfeatures) {
-    super(name, timeName, timeUnit, altName, altUnits);
+  public ProfileFeatureImpl(String name, List<CoordinateAxis> coords, double lat, double lon, double time,
```

Comment:
Can we make this `protected` instead of `public` maybe? If so, same question for `StationProfileCollectionImpl`, `StationProfileFeatureImpl`, and others with public constructors. Though they already were `public`, so we don't necessarily need to fix that in this PR, but I am just thinking after the next release that would technically be a breaking change.